### ISSUE TITLE
Remove the duplicate of IPN

### ISSUE THREAD

**Issue by vocna** [thetwom/Tuner#63]
I assume that _Default (A-B-C-...)_ and _International (A-B-C-...)_ are the same and it's confusing. 

Please, remove the duplicate and go to this:
> Carnatic
> Hindustani
> International (Default)
> Solfège

Current UI:
![Screenshot_20231006-152441](https://github.com/thetwom/Tuner/assets/147154374/6b81d4da-7cea-4066-8158-43aeb97d89b8)

**Comment by thetwom** [comment]
You are right in a sense here. Default and International are indeed the same for English language, but they are not for others (where one uses H instead of B). But ideally, we would have only one option, if they are the same to avoid confusion. Must see if I can change this ...
